tools/tests/test_check_constitution.py

```
@@ -0,0 +1,224 @@
+           "**Statute delta:** none.\n")
+    _write(r / "docs/architecture/constitution.md",
+           "# The constitution\n\n## 1. Identity\n\n- **A rule.** Its reason. [ADR-001:7]\n")
+    _run("git", "add", "-A", cwd=r)
+    _run("git", "commit", "-qm", "base", cwd=r)
+    _run("git", "branch", "base-ref", cwd=r)
+
+    monkeypatch.setattr(cc, "ROOT", r)
+    return r
+
+
+def _findings(repo: Path, pr: int | None = None) -> list[str]:
```

Ruff (0.16.1)

[warning] Unused function argument: `repo`

(ARG001)